pyproject.toml

```diff
@@ -71,7 +71,7 @@ tbats = "^1.1.0"
 Deprecated = "1.2.13"
 types-Deprecated = "1.2.9"
 
-prophet = {version = "^1.0", optional = true}
+prophet = {version = "^1.1", optional = true}
```

Comment:
What will happen if we remain `1.0` here? May be we can let old version exist if on clean environments new version is installed.